File: LeeLibrary/scroll/src/main/java/jv/lee/scroll/adapter/SimpleAdapter.java
package jv.lee.scroll.adapter;

import android.content.Context;

import com.lee.library.adapter.LeeViewAdapter;

import java.util.List;

/**
 * @author jv.lee
 * @date 2019-09-24
 * @description
 */
public class SimpleAdapter extends LeeViewAdapter<String> {

    public SimpleAdapter(Context context, List<String> data) {
        super(context, data);
        addItemStyles(new SimpleItem());
        addItemStyles(new SimpleListItem());
    }

}

File: KotlinLibrary/app/src/main/java/com/lee/app/view/adapter/SimpleAdapter.kt
package com.lee.app.view.adapter

import android.content.Context
import com.lee.library.adapter.LeeViewAdapter

/**
 * @author jv.lee
 * @date 2019/9/17.
 * @description
 */
class SimpleAdapter(context: Context, data: List<Int>) : LeeViewAdapter<Int>(context, data) {

    init {
        addItemStyles(SimpleItem())
    }

}